pkgs/development/python-modules/textual-autocomplete/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  textual,
+  typing-extensions,
+}:
+
+buildPythonPackage {
+  pname = "textual-autocomplete";
+  version = "3.0.0a9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "darrenburns";
+    repo = "textual-autocomplete";
+    rev = "bbacfa91bfd9ff006dab8930a8a3fe4ba46853ab";
```

Comment:
Mayybe add a comment explaining that this commit target the `version-two` branch